File: src/app/bootstrap-skynet.ts
/**
 * Skynet 环境入口文件
 * 初始化 Skynet 运行时并启动服务
 */

import { setRuntime } from '../framework/core/interfaces';
import { createSkynetRuntime } from '../framework/runtime/skynet-adapter';

// 初始化 Skynet 运行时
setRuntime(createSkynetRuntime());

// 预加载服务模块（可选，用于热更新）
require('./services/gateway/index');
require('./services/login/index');
require('./services/game/index');

// Skynet 服务会通过 skynet.start() 在各自的文件中启动

File: docker/lua/app/bootstrap-skynet.lua
local ____lualib = require("lualib_bundle")
local __TS__SourceMapTraceBack = ____lualib.__TS__SourceMapTraceBack
__TS__SourceMapTraceBack(debug.getinfo(1).short_src, {["5"] = 6,["6"] = 6,["7"] = 7,["8"] = 7,["9"] = 13,["10"] = 16});
local ____exports = {}
local ____interfaces = require("framework.core.interfaces")
local setRuntime = ____interfaces.setRuntime
local ____skynet_2Dadapter = require("framework.runtime.skynet-adapter")
local createSkynetRuntime = ____skynet_2Dadapter.createSkynetRuntime
setRuntime(createSkynetRuntime())
_G.require("app.services.gateway.index")
return ____exports
